src/interface/actions/reportHistory.ts

```diff
@@ -1,5 +1,20 @@
+export interface ReportHistory {
+  code: string;
+  title?: string;
+  start?: number;
+  end: number;
+  fightId?: number;
+  fightName?: string;
+  playerId?: number;
+  playerName: string;
+  playerRealm?: string;
+  playerRegion?: string;
+  playerClass: string;
+  type: number;
+}
+
 export const APPEND_REPORT_HISTORY = 'APPEND_REPORT_HISTORY';
-export function appendReportHistory(report: any) {
+export function appendReportHistory(report: ReportHistory) {
```

Comment:
```suggestion
export function appendReportHistory(report: ReportHistoryEntry) {
```

minor type name change for clarity---this type is not the history, but an *entry* in the history.